store/confirmCDS.go

```diff
@@ -57,24 +63,22 @@ func (m *mongoDB) CreateCDSData(result []schema.CDSData, country string) error {
 	if err != nil {
 		if errs, hasErr := err.(mongo.BulkWriteException); hasErr {
 			if 1 == len(errs.WriteErrors) && DuplicateKeyCode == errs.WriteErrors[0].Code {
+				log.WithFields(log.Fields{"prefix": mongoLogPrefix, "err": errs}).Warn("createCDSData Insert data")
 				fmt.Println(err)
```

Comment:
Thought not in the scope this PR, I came across this unnecessary print `fmt.Println(err)`.